tests/cpp_unit_tests/test_pgm_vnf_converter.cpp

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp>
+
+#include <doctest/doctest.h>
+
+namespace power_grid_model_io_native {
+
+TEST_CASE("Test converter constructor") { CHECK_THROWS_AS(PgmVnfConverter(), power_grid_model::ExperimentalFeature); };
```

Comment:
Please test the deletion as well.

Reply:
I think we would bump into the same issue as with other tests. If we can't create the constructor object then we definitely can't delete it